pkg/migration/controllers/migrationschedule.go

```diff
@@ -257,6 +270,31 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 
 		// Start a migration for a policy if required
 		if start {
+			// Reconciling the migrationschedule in remote cluster if there is a change
+			// Doing it here to limit the reconcilation check frequency
+			if remoteMigrSched != nil {
+				updatedHash, err := m.reconcileRemoteMigrationSchedule(migrationSchedule, remoteMigrSched)
+				if err != nil {
+					msg := fmt.Sprintf("Error reconciling remote migration schedule: %v", err)
+					log.MigrationScheduleLog(migrationSchedule).Error(msg)
+					return fmt.Errorf(msg)
+				}
+				if updatedHash != 0 {
+					// update the hash value in the source migration schedule
```

Comment:
nit...I think you mean remote migration schedule in the comments here?

Reply:
Done